gen/definitions/router_syslog_vrf.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: syslog_vrf
+path: Cisco-IOS-XR-um-logging-cfg:/logging/vrfs/vrf[vrf-name=%s]
+doc_category: logging
+attributes:
+  - yang_name: vrf-name
+    id: true
+    example: CORE-Mgmt
+
+  - yang_name: host-ipv4-addresses/host-ipv4-address
```

Comment:
When we add this for 'host-ipv4-addresses', let's add the same for ipv6 and names as well.

Reply:
support added for both ipv4 & ipv6